nntrainer/layers/embedding.cpp

```diff
@@ -23,10 +23,36 @@ namespace nntrainer {
 
 const std::string EmbeddingLayer::type = "embedding";
 
+enum EmbeddingParams { weight };
+
 int EmbeddingLayer::initialize(Manager &manager) {
   int status = ML_ERROR_NONE;
+  if (num_inputs != 1) {
+    throw std::invalid_argument("Embedding layer takes only one input");
+  }
 
-  // NYI
+  output_dim[0] = input_dim[0];
+
+  output_dim[0].height(in_length);
+  output_dim[0].width(out_dim);
+  input_dim[0].width(in_length);
+
+  TensorDim dim = output_dim[0];
+
+  dim.height(in_dim);
+  dim.width(out_dim);
+  dim.batch(1);
```

Comment:
I have a small code clean suggestion :)
 `TensorDim dim({in_dim, out_dim});` or `TensorDim dim({output_dim.channel(), in_dim, out_dim});`

Reply:
I wonder if `channel() != 1` it should be error or okay?

Reply:
for now, channel need to be 1. I will add check.